CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -104,16 +117,30 @@ private JObject RetrieveFromEndPoint(string address)
 
         private JObject RetrieveWithPagination(string address)
         {
-            var apiResponse = RetrieveFromEndPoint($"{address}&page=1");
+            var apiResponse = RetrieveFromEndPoint(AppendURLWithPageNumber(address, 1));
+            
+            if (apiResponse["total_pages"] == null) {
+                Console.WriteLine(apiResponse);
```

Comment:
Where does the output of `Console.WriteLine` go when this code is running on Heroku?

Reply:
Very good point!  Removed 👍